src/index.js

```diff
@@ -26,6 +26,7 @@ export default class KintoApi {
    * - {EventEmitter} events      The events handler. If none provided an
    *                              `EventEmitter` instance will be created.
    * - {Object}       headers     The key-value headers to pass to each request.
+   * - {Boolean}      safe        Perform safe requests (default: `false`).
```

Comment:
I think we'll have to create proper human readable for this project, esdocs are not enough I'm afraid. See #16.